packages/platform-ios/src/link-pods/__tests__/isInstalled.test.ts

```diff
@@ -18,18 +16,21 @@ describe('pods::isInstalled', () => {
   it('returns false if pod is missing', () => {
     const project = {podfile: path.join(PODFILES_PATH, 'PodfileSimple')};
     const podspecName = {podspecPath: '/path/NotExisting'};
+    // @ts-ignore FIXME: Improve types
```

Comment:
What was the issue with these?

Reply:
The `isInstalled` function is not written correctly. We are passing whole big objects to the function, but not all properties are used. I've added `ts-ignore` in the test to prevent adding unused mock data.